pkg/template/apis/template/validation/validation.go

```diff
@@ -71,15 +70,6 @@ func ValidateTemplateInstance(templateInstance *templateapi.TemplateInstance) (a
 			allErrs = append(allErrs, field.Required(field.NewPath("spec.secret.name"), ""))
 		}
 	}
-	if templateInstance.Spec.Requester == nil {
-		allErrs = append(allErrs, field.Required(field.NewPath("spec.requester"), ""))
-	} else if templateInstance.Spec.Requester.Username == "" {
-		allErrs = append(allErrs, field.Required(field.NewPath("spec.requester.username"), ""))
```

Comment:
ok, capturing here another off line converation @bparees and I had ... the key points:

- given @bparees point immediately preceding this one, added these username checks back in
- we also talked to @deads2k about group/extra, and the net was they could be anything (think certs scenario wrt groups), so we are punting on adding validation for those new fields